packages/server/test/typings-test.ts

```diff
@@ -16,13 +17,20 @@ adapter(http2Req, http2Res);
 const http2Server = createHttp2Server(adapter);
 http2Server.on('request', adapter);
 
-const httpReq = null as unknown as IncomingMessage;
+const httpReq = null as unknown as NodeRequest;
```

Comment:
We want to test our adapter with `IncomingMessage`. This defeats the goal of the test.